build/Tests/FunctionalTests/FunctionalTests.vcxproj

```diff
@@ -281,6 +281,7 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\tests\functionaltests\Tests\NSURLStorageFileTests.mm">
       <ObjectiveCARC>true</ObjectiveCARC>
     </ClangCompile>
+    <ClangCompile Include="$(WINOBJC_SDK_ROOT)\tests\unittests\CoreText\CTRunTests.mm" />
```

Comment:
><ClangCompile Include="$(WINOBJC_SDK_ROOT)\tests\unittests\CoreText\CTRunTests.mm" /> [](start = 4, length = 85)

Shows a "fancy" unit test working. Was added here since FunctionalTests rely on a custom logger for TAEF that will need injected into other builds via .props / .targets but then we can move UTs to use TAEF.